packages/server/src/config.ts

```diff
@@ -48,27 +59,44 @@ const defaults = {
     "cypress/**/*",
   ],
   includePaths: ["**/*"],
-  devFolder: ".blitz/caches/dev",
-  buildFolder: ".blitz/caches/build",
-  nextBinPatched: "./node_modules/.bin/next-patched",
-  writeManifestFile: true,
 }
 
-export async function normalize(config: ServerConfig): Promise<NormalizedConfig> {
-  const nextBinOrig = await resolveBinAsync("next")
-  const nextBinPatched = await resolveBinAsync("@blitzjs/server", "next-patched")
-  const git = parseChokidarRulesFromGitignore(resolve(process.cwd(), config.rootFolder))
+export async function normalize(
+  config: ServerConfig,
+  dev: boolean = false,
```

Comment:
Any reason why not just add `dev` to `ServerConfig`?

Reply:
There is no special reason for this. It could definitely be inside `ServerConfig`. In that case I suggest to use it like following:

```js
export type ServerConfig = {
  env: "dev" | "prod"
  ...
}
```

While 'prod' is default. What do you think?

Reply:
Works for me

Reply:
Done in current commit